### ISSUE TITLE
Suggestion: mass sales

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#59]
```
Suggestion: when selling an item, if you have more than one (for example Meat 
x19), a popup with amount you want to sell. In addition, there also could be 
item filters or something like that (All, weapons, potions, others etc.), to 
show only items in this category.
```

Original issue reported on code.google.com by `rafal.wa...@gmail.com` on 3 Dec 2010 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
The same for dropping unwanted and unsellable items!
```

Original comment by `erwr...@gmail.com` on 3 Dec 2010 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Good idea! This is something that would make highlevel play less tedious.

It could also be merged with a dialog box discussed in issue 51 (confirm 
sell/buy).
```

Original comment by `oskar.wi...@gmail.com` on 3 Dec 2010 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
I agree, either a slider bar pop up or a text box where you can type in the 
amount you want to sell.
```

Original comment by `sdeva...@gmail.com` on 9 Feb 2011 at 11:15
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Instead of a slider, perhaps having a long hold, while in the menu, to sell 
all. 
```

Original comment by `medic....@gmail.com` on 1 Jun 2011 at 6:05

**Comment by GoogleCodeExporter** [comment]
```
If this were happen they would have to do a confirm option in case your device 
holds on to item when you are just clicking on the item. We could also put this 
under preferences whether you want do mass sell or sell all like you suggested.
```

Original comment by `dope...@gmail.com` on 1 Jun 2011 at 7:33

**Comment by GoogleCodeExporter** [comment]
```
Should be merged with issue 51:
http://code.google.com/p/andors-trail/issues/detail?id=51
```

Original comment by `SamuelPl...@gmail.com` on 9 Jul 2011 at 6:26

**Comment by GoogleCodeExporter** [comment]
```
With the excellent work done by ejwessel, this is now completed in r131. Now an 
interface pops up requesting you to enter the quantity of items to act upon. 
The dialog also has [+1]/[-1] buttons and a slider where you can select the 
amount. 

This applies to dropping, buying and selling.

Marking this as ready for the next release.
```

Original comment by `oskar.wi...@gmail.com` on 15 Jul 2011 at 8:28
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Also thanks to Samuel.
```

Original comment by `sdeva...@gmail.com` on 15 Jul 2011 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
I can see how this could help if you were selling like a 100 items but so far I 
like the current quick version at my low level. Can oldschool be retained?
```

Original comment by `nezkeys1...@gmail.com` on 4 Sep 2011 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
